WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -363,7 +363,7 @@
                     android:id="@+id/my_site_sharing_icon"
                     style="@style/MySiteListRowIcon"
                     android:importantForAccessibility="no"
-                    android:src="@drawable/my_site_icon_sharing"/>
+                    android:src="@drawable/ic_share_grey_dark_24dp"/>
```

Comment:
This should use `app:srcCompat="@drawable/ic_share_grey_dark_24dp"` rather than `android:src="@drawable/ic_share_grey_dark_24dp"`.

Reply:
Thanks! Changed.